__tests__/serverSpec.jsx

```diff
@@ -82,7 +82,7 @@ describe('post /', () => {
   test('responds with a 201 to a post request', () => {
     supertest(app)
       .post('/')
-      .field(35, 'my awesome avatar')
+      .field('chinese', '205')
```

Comment:
This test would still pass and I am thinking it's because this is an asynchronous request and jest ended the test beforehand. Is this correct?